tools/run_tests.py

```diff
@@ -148,9 +148,7 @@ def run_shakedown(self, test_dirs, requirements_txt=None, pytest_types='sanity')
 #!/bin/bash
 set -e
 echo "VIRTUALENV CREATE/UPDATE: {venv_path}"
-if  [ ! -f {venv_path}/bin/python3 ] ; then
-    virtualenv -p $(which python3) --always-copy {venv_path}
-fi
+virtualenv -p $(which python3) --always-copy {venv_path}
```

Comment:
the if was a failed idea as to how to safely parallelize.  In this revert it's going back to how it was, but this idea didn't work anyway as pip would uninstall packages out from other another test run.